src/ng/compile.js

```diff
@@ -2222,13 +2235,15 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
           }
           if (slotName) {
             var slotTranscludeFn = boundTranscludeFn.$$slots[slotName];
-            if (!slotTranscludeFn) {
+            if (slotTranscludeFn) {
+              return slotTranscludeFn(scope, cloneAttachFn, transcludeControllers, futureParentElement, scopeToChild);
+            }
+            if (isUndefined(slotTranscludeFn)) {
```

Comment:
We have to differentiate between `null` and `undefined`. I will add a comment to clarify.